README.md

```diff
@@ -144,6 +180,18 @@ haven't already:
   ],
 ```
 
+### With another Jest-compatible `expect`
```

Comment:
Would this be the approach if you're not using vitest's globals.

Reply:
I'm not 100% sure what you mean by vitest's globals. Vitest's default mode is `globals: false`, which requires explicit imports of describe, expect, etc., and the new vitest entrypoint is designed to work with that. I didn't actually think to test with `globals: true` and importing the `vitest/globals` types &mdash; I'll do that tomorrow.

This section was intended as a hedge against future testing libraries that might emerge that retain some level of compatibility with Jest. But if you think it might be more confusing than helpful, I'm happy to remove it.

Reply:
> I didn't actually think to test with `globals: true` and importing the `vitest/globals` types — I'll do that tomorrow.

Tested this and it works.